Source/WebCore/layout/formattingContexts/grid/ImplicitGrid.h

```diff
@@ -55,6 +64,8 @@ class ImplicitGrid {
     void insertItemInArea(const UnplacedGridItem&, size_t columnStart, size_t columnEnd, int rowStart, int rowEnd);
 
     GridMatrix m_gridMatrix;
+    int m_explicitGridRowStart { 0 };
+    int m_explicitGridColumnStart { 0 };
 };
```

Comment:
If I understand the spec correctly, we should be able to make these unsigned right? The location of the explicit grid within the implicit should be some non-negative value I think?